### PR TITLE
Login/Logout Glyph

### PR DESCRIPTION
**Pull request by Amstutz**
Introduces the needed Login/Logout Glyph, see: https://mantis.ilias.de/view.php?id=26663

image

image

### CODE REVIEW

**Review by yvseiler — CHANGES_REQUESTED** [review]
Hi @Amstutz 

Thank you a lot for contributing to ILIAS.

Clear description and differentiation from Logout Glyph, thanks a lot. However, there are still two copy-paste errors from my point of view.

Please implement the following changes:
**Login-Glyph description:**
- [x] Change "Logout Glyph" to "Login Glyph" in section "composition"

**Logout-Glyph description:**
- [x] Change "The Logout Glyph" to "The Login Glyph" in section "rivals"


[optional_comments_on_how_to_proceed]

kindly,
@yvseiler

File: src/UI/Component/Symbol/Glyph/Factory.php [hunk 1]
```diff
@@ -1065,4 +1065,62 @@ public function more($action = null);
...
+     *       Clicking this Glyph will trigger the interaction to authenticate and login.
+     *   rivals:
+     *       Logout Glyph: The Logout Glyph triggers the logout interaction.
+     *
+     * context:
+     *    - The Login Glyph appears in the Meta Bar.
+     *
+     * rules:
+     *   usage:
+     *       1: The Login Glyph MUST be displayed of no user is authenticated.
```
Comment (yvseiler) [inline src/UI/Component/Symbol/Glyph/Factory.php]: "if" instead of "of"?

**Review by yvseiler — APPROVED** [review]
Hi @Amstutz 

Thanks a lot! No further changes requested.

kindly,
@yvseiler

**Review by klees — APPROVED** [review]

**Comment by matthiaskunkel** [comment]
Jour Fixe, 02 DEC 2019 : PR highly appreciated for ILIAS 6. Please merge to trunk.

**Pull request merged**